documentation/modules/managing/proc-manual-rolling-update-statefulset.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// managing/assembly-rolling-updates.adoc
+
+[id='proc-manual-rolling-update-statefulset-{context}']
+= Performing a rolling update using a StatefulSet annotation
+
+This procedure describes how to manually trigger a rolling update of an existing Kafka cluster or ZooKeeper cluster using a Kubernetes `StatefulSet` annotation.
+
+.Procedure
+
+. Find the name of the `StatefulSet` that controls the Kafka or ZooKeeper pods you want to manually update.
```

Comment:
The recommended style is StatefulSet without the monospace, unless you are referring to a specific resource. For example: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/

Reply:
We're referring to resources